custom-commands/inotify-proxy/README.md

```diff
@@ -0,0 +1,11 @@
+# inotify-proxy to enable file watchers on NFS shares
+
+This tools helps to detect changed files in Docker Containers if NFS mound is used.
```

Comment:
Please say where the custom command is to be installed.

Reply:
It's not clear reading the inotify-proxy project page whether this is a tool that runs on the host side or the container side. Or if it's linux only for the host side. If it's linux-only, please clear that up there and here. Also, if it's linux-only, could you add an arm64 build?

Reply:
Right... I am a bit professionally blinkered. I will update the docs.
It's running inside the container. I can add an ARM build but I have no device to test it.

Reply:
Added additional description and released v2.1.0 of inotify-proxy with linux arm64 support.